autovalet_navigation/config/sim/base_local_planner_params.yaml

```diff
@@ -78,11 +78,11 @@ TebLocalPlannerROS:
  weight_acc_lim_x: 1.0
```

Comment:
maybe don't push this file? we'll have a better version when i eventually merge my `kevin/planner_2` branch with the best param list

Reply:
sure I will revert it to its original version